specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json

```diff
@@ -1827,18 +1818,9 @@
           "description": "/29 IP address space to carve out Customer addresses for tunnels."
         },
         "circuitConnectionStatus": {
-          "type": "string",
-          "description": "Express Route Circuit Connection State. Possible values are: 'Connected' and 'Disconnected'.",
-          "enum": [
-            "Connected",
-            "Connecting",
-            "Disconnected"
-          ],
           "readOnly": true,
```

Comment:
The "readOnly" field should be moved out from here and placed in the definition. Every instance has "readOnly" = true.